tests/spark.py

```diff
@@ -1,3 +1,7 @@
+import os
 from pyspark.sql import SparkSession
 
-spark = SparkSession.builder.master("local").appName("chispa").getOrCreate()
+if "SPARK_CONNECT_MODE_ENABLE" in os.environ:
```

Comment:
This variable is `SPARK_CONNECT_MODE_ENABLED` in [spark](https://github.com/apache/spark/blob/95de02e80deef19607a55b5913c674cc21521132/python/pyspark/sql/session.py#L490), let's use the same name here.